src/clientmetricreport/ClientMetricReport.ts

```diff
@@ -37,7 +37,7 @@ export default class ClientMetricReport {
   };
 
   decoderLossPercent = (metricName?: string, ssrc?: number): number => {
-    const metricReport = ssrc ? this.streamMetricReports[ssrc] : this.globalMetricReport;
+    const metricReport = this.streamMetricReports[ssrc];
```

Comment:
This is only for `streamMetricReport`, and the metric does not exist in `globalMetricReport`, so removed the code to reduce confusion.

Reply:
We have a snippet in our [Quality, Bandwidth, and Connectivity](https://github.com/aws/amazon-chime-sdk-js/blob/main/guides/04_Quality_Bandwidth_Connectivity.md) guide that we can update. I think it would be good to add some more details on how it works, but I will be reworking this guide in my upcoming change if we wanna wait till then.